app/models/account.rb

```diff
@@ -562,8 +562,9 @@ def provider_id_for_audits
     end
   end
 
+  # TODO: THREESCALE-3759 remove this method
   def independent_mapping_rules_enabled?
-    provider_can_use?(:api_as_product) || provider_can_use?(:independent_mapping_rules)
+    true
   end
```

Comment:
You can directly remove it in https://github.com/3scale/porta/pull/2274#issuecomment-739999173 and perhaps that PR point to `master` directly